### PR TITLE
Completed es6-const-let.

### PR DESCRIPTION
**Pull request by Youssef-Najjarine**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
```
Comment (BrettAlbright) [inline es6-const-let/index.html]: This should be a `const`

File: es6-const-let/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
```
Comment (BrettAlbright) [inline es6-const-let/index.html]: This should be a const

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just a couple changes to const. We want to use const for all variables which contain arrays and objects, and for variables where the value assigned will not change

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**